CHANGELOG.md

```diff
@@ -67,11 +67,11 @@ You can find all deprecations in [this overview](https://github.com/kedacore/ked
 
 New deprecation(s):
 
-- **General**: Clean up previously deprecated code for 2.12 release ([#4899](https://github.com/kedacore/keda/issues/4899))
+- TODO ([#XXX](https://github.com/kedacore/keda/issues/XXX))
 
 ### Breaking Changes
 
-- TODO ([#XXX](https://github.com/kedacore/keda/issues/XXX))
+- **General**: Clean up previously deprecated code for 2.12 release ([#4899](https://github.com/kedacore/keda/issues/4899))
```

Comment:
Is that really a deprecation? Isn't that just cleaning up? Or you mean that we're deleting things that were announced to be deprecated? If so, then we need to list the things that are removed.

Reply:
We removed the support to `metricName` following the deprectaion policy. It was deprecated with a warningin v2.10 and v2.12 removes the support, so I think it's a breaking change because users with any stuff based on it will have to do manual interaction (although we announced 2 versions ago)

Reply:
That's fine! But let's document that instead of "deprecated code etc etc". ENd-users should be able to assess impact just by checking this section on its own imo

Reply:
done, PTAL @tomkerkhove

Reply:
A lot better, thanks!